kubeconfig_test-managed-control-plane

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
Where this file used? cannot find

Reply:
Oh I have no idea what this is either, I will fix up my PR to remove this.